tools/integration_tests/util/operations/file_operations.go

```diff
@@ -192,3 +194,38 @@ func ReadFileSequentially(filePath string, chunkSize int64) (content []byte, err
 	}
 	return
 }
+
+func WriteFileSequentially(filePath string, fileSize int64, chunkSize int64) (err error) {
+	var numberOfBytes int
```

Comment:
Why do we need to declare the variable on the top? We should always initialise, where the variable will be used.